modules/govuk/manifests/apps/email_alert_api.pp

```diff
@@ -232,4 +232,9 @@
       }
     }
   }
+  else {
+    govuk::app { 'email-alert-api':
+      ensure => 'absent',
```

Comment:
I think you can use the `absent` keyword here rather than a string.

Reply:
I thought I would leave it consistent with the other occurrence in the file.

Reply:
Yeah, I don't mind either way!